contracts/ronin/validator/storage-fragments/ValidatorInfoStorage.sol

```diff
@@ -222,7 +222,7 @@ abstract contract ValidatorInfoStorage is IValidatorInfo, HasRoninTrustedOrganiz
    * @dev See `IValidatorInfo-setMaxPrioritizedValidatorNumber`
    */
   function _setMaxPrioritizedValidatorNumber(uint256 _number) internal {
-    if (_number > _maxValidatorNumber) revert ErrInvalidMaxPrioritizedValidatorNumber();
+    if (_number > _maxValidatorNumber) revert ErrInvalidMaxPrioritizedValidatorNumber(msg.sig);
```

Comment:
Why should need a `msg.sig` here? Are there more than 1 method using this error?

Reply:
removed msg.sig